src/Command/MeilisearchImportCommand.php

```diff
@@ -48,7 +49,6 @@ public static function getDefaultDescription(): string
     protected function configure(): void
     {
         $this
-            ->setDescription(self::getDefaultDescription())
```

Comment:
Was it removed because it already contains the `default description`?

Reply:
It is related to https://github.com/rectorphp/rector-symfony/blob/main/docs/rector_rules_overview.md#commanddescriptiontopropertyrector

Reply:
In our case we don't have it as a property, it will work the same way right?

Reply:
Yes but with deprecation triggedered
Or we should directly rely on the static prop
I am open to suggestion here

Reply:
Copy that
Afk for 2 days but will come back to this shortly